daemon/daemon_unix.go

```diff
@@ -189,6 +197,15 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *runconfig.HostC
 			return warnings, fmt.Errorf("Invalid value: %v, valid memory swappiness range is 0-100.", swappiness)
 		}
 	}
+	if hostConfig.KernelMemory > 0 && !sysInfo.KernelMemory {
+		warnings = append(warnings, "Your kernel does not support kernel memory limit capabilities. Limitation discarded.")
```

Comment:
*discard* means to throw away.  Do you mean ignored?  It looks like the user specified a limit for a kernel that does not support memory limits.

"You specified a memory limit which will is ignored because your kernel does not have memory limit capabilities."